store-implementation/proxy-store/src/main/java/uk/gov/gchq/gaffer/proxystore/ProxyStore.java

```diff
@@ -322,6 +336,19 @@ protected void addAdditionalOperationHandlers() {
         addOperationHandler(OperationChain.class, new OperationChainHandler<>(opChainValidator, opChainOptimisers));
         addOperationHandler(OperationChainDAO.class, new OperationChainHandler<>(opChainValidator, opChainOptimisers));
         addOperationHandler(GetTraits.class, getGetTraitsHandler());
+
+        if (nonNull(CacheServiceLoader.getService())) {
```

Comment:
So now the ProxyStore explicitly enables it's own cache operations, like AddNamedOperation, in case you want to store named ops on the proxy? I guess you might want to do that but it may get a little confusing, in this case how would I instead choose to send on the named operation to the target store?

Reply:
gets done here https://github.com/gchq/Gaffer/pull/2921